docs/inputs.md

```diff
@@ -1319,7 +1031,6 @@ regional_pass: 10.90
     
 </table>
 
-#### CVM Establishment Synthesis File ( MGRAEmpByEstSize.csv )
 ##### `MGRAEmpByEstSize.csv`
```

Comment:
is this file under subheader Commercial vehicle model? please double check @mansisandag

Reply:
Creating another PR because of branch issues to address this.